taroscript/send.go

```diff
@@ -254,6 +254,105 @@ func SignVirtualTransaction(input *taropsbt.VInput, outputs []*taropsbt.VOutput,
 	return nil
 }
 
+// CreateOutputCommitments creates the final set of TaroCommitments representing
+// the asset send. The input TaroCommitment must be set.
+func CreateOutputCommitments(inputCommitment *commitment.TaroCommitment,
+	input *taropsbt.VInput,
```

Comment:
Seems like we can actually _embed_ `inputCommitment` within `input`?

Reply:
I intended to do that initially. But then I figured that the commitment might be enormous, depending on how many assets and asset coins might be contained in it. So I opted to _not_ include that in the packet. Since it's not needed during the signing process of the virtual transaction, this should be fine. But it does mean that the entity that creates the anchor TX must be able to fetch the full input commitment from somewhere. So currently only the owner of the spent asset could do it. But I think with universes being present this information could also be fetched from somewhere.